src/components/Music/Music.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import style from './Music.module.css'
```

Comment:
You forgot a new line above style import.

Reply:
/fxd